### PR TITLE
[api-extractor] Add support for a new AEDoc tag "@eventproperty"

### PR DESCRIPTION
**Pull request by pgonzal**
We recently proposed a new TSDoc tag `@eventProperty` for documenting class/interface properties that represent events.  The design was discussed here: https://github.com/Microsoft/tsdoc/issues/30

*NOTE: The upcoming TSDoc standard will recommend camel case tags (e.g. `@eventProperty`). However currently API Extractor uses lowercase (e.g. `@eventproperty`).  Changing the casing convention is out of scope for this PR.*

### CODE REVIEW

**Comment by pgonzal** [comment]
@AlexJerabek @iclanton FYI

File: rush.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "rushVersion": "5.0.0-dev.24",
+  "rushVersion": "5.0.0",
```
Comment (pgonzal) [inline rush.json]: >5.0.0 [](start = 18, length = 5)

(I also upgraded Rush in this PR)

**Comment by iclanton** [comment]
Approved

**Comment by pgonzal** [comment]
@AlexJerabek can you confirm that this is working now? Should we merge it?

**Comment by AlexJerabek** [comment]
@pgonzal @iclanton This is working end-to-end. Thanks! Please check it in.

**Pull request merged**